### PR TITLE
Update company.ru.robomarkup

### PR DESCRIPTION
**Pull request by HSteinway**
Translated.

### CODE REVIEW

File: company/company.ru.robomarkup [hunk 1]
```diff
@@ -11,85 +11,85 @@
...
 
 
 #!company id{c-ack}
-Acknowledge
+Подтверждение
 
 
 #!company id{c-arrive}
-# Re: Iceni salvage arrival report
+# Re: Отчет о прибытии специалиста на Льдынь
```
Comment (alexheretic) [inline company/company.ru.robomarkup:28]: "Iceni" is the name of the space ship. Just checking if this translation makes sense in that context?

File: company/company.ru.robomarkup [hunk 2]
```diff
@@ -11,85 +11,85 @@
...
 
-Can't wait to tell Nigel. Ooh... maybe it's an Iris thing. Or even a Martin thing! You find something juicy and I could be lunching on Titan with big M before the end of the fiscal year. Go team!
+Не могу дождаться, чтобы рассказать Найджелу! Ох... может, еще Айрис расскажу. Или даже Мартину! Ты найди там что-нибудь сочненькое, и я смогу обедать с большим М. на Титане еще до конца отчетного года. Вперед, команда!
 
-Sometimes I envy you, you know. Out there, thousands of miles from home with no distractions from family, friends or leisure activities. Completely alone with your thoughts. Many would kill for the kind of inner peace I imagine you experience.
+Иногда я завидую тебе, знаешь. Где-то там, в тысячах километров от дома, и ни семья, ни друзья, ни всякие очень приятные дела тебя не отвлекают. Наедине со своими мыслями. Многие бы убили за возможность найти тот внутренний покой, который тебе дарован.
 
-Keep up the adequate work. And don't accept any attitude from that robot.
+Продолжай в том же духе. И сильно не привязывайся к этому роботу.
```
Comment (alexheretic) [inline company/company.ru.robomarkup]: > don't accept any attitude from that robot.

This means don't take any back talk / disrespect from the robot, it's a bit of ludicrous management advice. Did that come accross in this translation?

**Comment by alexheretic** [comment]
Fantastic work! You are really smashing out these translations thank you!

Just a couple of questions but this looks very good to me :+1:

File: company/company.ru.robomarkup (hunk 1 above)
Comment (HSteinway) [reply company/company.ru.robomarkup:28]: Yeah, I got it. "Льдынь" is a kinda great piece of ice, but it's also a non-existing word. There's a mention in a message, that it's a space ship. If you mean by Iceni something specific, just tell me.

File: company/company.ru.robomarkup (hunk 2 above)
Comment (HSteinway) [reply company/company.ru.robomarkup]: That's something I didn't get) I'll fix it.

**Comment by alexheretic** [comment]
Nice one. I'll upload some in game renders for these later today when I'm back at the keyboard.

**Comment by alexheretic** [comment]
Sorry for the delay here, had to do a little more as the company font didn't support cyrillic so I've swapped it out for one that does when needed.

company-1
company-2
company-3

**Comment by HSteinway** [comment]
Nice. It'll help when we're up to translate the levels names. Where are they btw?)

**Comment by alexheretic** [comment]
The level names aren't actually available to translate at the moment. I guess I thought of them like the game name, ie not something to translate.

But I don't see a problem with translating them. I just need to extract them to a file here somewhere.

**Comment by HSteinway** [comment]
I guess, they could be translated) We'll see. I'm in the middle of the colony.ru.robomarkup translation. Got some questions. For ex. what is 'assembly-17' in: I am making available the Engineering Primer I've somewhat hastily constructed to accompany the workshops following assembly-17.19.12.2019, 00:13, "Alex Butler" <notifications@github.com>:The level names aren't actually available to translate at the moment. I guess I thought of them like the game name, ie not something to translate.
But I don't see a problem with translating them. I just need to extract them to a file here somewhere.

**Comment by HSteinway** [comment]
I guess, they could be translated) We'll see. I'm in the middle of the colony.ru.robomarkup translation. Got some questions. For ex. what is 'assembly-17' in:
I am making available the Engineering Primer I've somewhat hastily constructed to accompany the workshops following assembly-17.

**Comment by alexheretic** [comment]
"Assembly-17" was the 17th meeting of the colonists to discuss the plight they faced after landing on the planet Bright.

Great to hear you're looking at the colony messages! You're the first to try translating them so please ask about anything. After I'll try to add clarifications to the translation base.

**Pull request merged**